lib/product-import.coffee

```diff
@@ -47,7 +50,7 @@ class ProductImport
 
   _resetCache: ->
     @_cache =
-      productType: {}
+      productType: {} # productTypeId : productType map
```

Comment:
Why not just calling the map what it does `productTypeId2productType` or shorter `id2productType` ?